install/automationservicebroker/install-rbac.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: automation-broker-apb-rbac
+objects:
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: automation-broker-apb
+  roleRef:
+    name: cluster-admin
```

Comment:
Please make these permissions match https://github.com/openshift/openshift-ansible/blob/master/roles/ansible_service_broker/tasks/install.yml#L30-L90 .

Reply:
hmmm... perhaps I misunderstood what's happening.  This is basically an installer pod?